extensions/audit-reports/manifest.json

```diff
@@ -39,5 +39,12 @@
     "unpublished.qmd": {
       "checksum": "dd2d6de0ce972611a454e80776c4ed3d"
     }
+  },
+  "extension": {
+    "name": "audit-reports",
+    "title": "Audit Reports",
+    "description": "A quarto site of audit reports.",
+    "homepage": "https://github.com/posit-dev/connect-extensions/tree/main/extensions/audit-reports",
+    "version": "0.0.0"
```

Comment:
Would it make sense for this to use a link that won't change if the extension is updated?

Reply:
I'm not sure I follow this question - can you expand a bit more?

Reply:
As in, would it make sense to use a permalink:

`https://github.com/posit-dev/connect-extensions/tree/5e5066d9b0ad5f5d3b1454a2cc2437cf97bad5c9/extensions/audit-reports`

rather than the link to `tree/main`, which may refer to something different in the future if the extensions is updated.

Reply:
Ah that is a great point. Currently the extension list doesn't capture a different homepage for each version instead having a homepage for the entire extension. This is similar in concept to how Homebrew has https://cli.github.com/ as the homepage for the `gh` formulae.

There could be a good amount of value in splitting that up per-version especially since the homepage is linking to a git repo (it doesn't have to).